doc/manpages/man1/afp_lantest.1.md

```diff
@@ -4,14 +4,13 @@ afp_lantest — AFP LAN performance and directory cache testing tool
 
 # Synopsis
 
-**afp_lantest** [-34567GgVvbC] [-h *host*] [-p *port*] [-s *volume*] [-u *user*] [-w *password*]
+**afp_lantest** [-34567GgVvbcK] [-h *host*] [-p *port*] [-s *volume*] [-u *user*] [-w *password*]
```

Comment:
Nitpick: Let's keep the letter options sorted in alphabetic order. I.e. `[-34567bcGgKVv]`

When I started maintaining the testsuite code, I found several cases of letter options being duplicated in both documentation and code, and spent considerable time getting it cleaned up and sorted. ;)

Reply:
Done